website/docs/reference/warehouse-profiles/impala-profile.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Apache Impala Profile"
+id: "impala-profile"
+---
+
+## Overview of dbt-impala
+
+**Maintained by:** Cloudera    
+**Author:** Cloudera    
+**Source:** [Github](https://github.com/cloudera/dbt-impala)    
+**dbt Cloud:** Currently un-supported    
+**dbt Slack channel** [Link to channel](https://slack.com)     
```

Comment:
doesn't have to happen in this PR, but I recommend asking for a `#db-impala` channel to be made in [#channel-requests](https://getdbt.slack.com/archives/C01D8J8AJDA).

Reply:
I have made a request to that channel, will update the docs when the same is created.